### PR TITLE
Eliminate constructors

### PR DESCRIPTION
**Pull request by driesd**
### Description
Eliminate all component constructors.

### Breaking changes
None

### CODE REVIEW

File: components/banner/Banner.js [hunk 1]
```diff
@@ -35,7 +26,7 @@ class Banner extends PureComponent {
               className={theme['close-button']}
               icon={<IconCloseMediumOutline />}
               color={this.getCloseButtonColor()}
-              onClick={this.handleClick}
+              onClick={onClose}
```
Comment (duivvv) [inline components/banner/Banner.js]: shoudn't this one be in the destructuring?

Comment (driesd) [reply components/banner/Banner.js]: I think this is already the case?

const { children, className, icon, `onClose`, fullWidth, ...others } = this.props;

Comment (InstaK) [reply components/banner/Banner.js]: @duivvv it is, you probably looked at the props of getCloseButtonColor? 😄

Comment (duivvv) [reply components/banner/Banner.js]: Ah, the diff got me confused :)

File: components/tooltip/Tooltip.js [hunk 2]
```diff
@@ -30,17 +30,12 @@ const SIZES = {
...
-      super(...arguments);
+    tooltipRoot = this.props.documentObject.createElement('div');
 
-      this.state = {
-        active: false,
-        position: this.props.tooltipPosition,
-      };
-
-      this.tooltipRoot = props.documentObject.createElement('div');
-      this.tooltipRoot.id = 'tooltip-root';
```
Comment (lowiebenoot) [inline components/tooltip/Tooltip.js]: This one wasn't used and could just be dropped I guess?

**Pull request merged**